ci/release/update-version.sh

```diff
@@ -75,8 +75,26 @@ sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/cudf/source/
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/dask_cudf/source/conf.py
 sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/dask_cudf/source/conf.py
 
+DEPENDENCIES=(
+  cudf
+  cudf_kafka
+  custreamz
+  dask-cuda
+  dask-cudf
+  kvikio
+  libkvikio
+  librmm
+  rmm
+)
+for DEP in "${DEPENDENCIES[@]}"; do
+  for FILE in dependencies.yaml conda/environments/*.yaml; do
+    sed_runner "/-.* ${DEP}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}\.*/g" ${FILE}
+  done
+  for FILE in python/*/pyproject.toml; do
+    sed_runner "s/${DEP}==.*\",/${DEP}==${NEXT_SHORT_TAG_PEP440}.*\",/g" ${FILE}
+  done
+done
```

Comment:
```suggestion
for DEP in "${DEPENDENCIES[@]}"; do
  for FILE in dependencies.yaml conda/environments/*.yaml; do
    sed_runner "/-.* ${DEP}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
  done
  for FILE in python/*/pyproject.toml; do
    sed_runner "/\"${DEP}==/ s/==.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" ${FILE}
  done
done
```

A few minor suggestions here:

- We can drop the `\` in line 91 since the replacement pattern is interpreted literally and not as a regex (I think I added this initially in another PR, my bad).
- For the `pyproject.toml` loop, we can use sed [address ranges](https://www.gnu.org/software/sed/manual/sed.html#Addresses-overview) like we did in the preceding `for` block to dedupe the command a bit (e.g. only specifiying `${DEP}` once).
- Also in the `pyproject.toml` loop, we can remove the `,` since it's common to both the search and replace pattern. The escaped quotations (`\"`) are sufficient for determining the end of the search and replacement strings.

Reply:
You sed it best! Applying your changes.